flags.go

```diff
@@ -106,11 +110,27 @@ func RegisterFlags() {
 	}
 }
 
+func EnvVarFlags() {
+	var value string
+
+	for f := range rtlamrFlags {
+		value = os.Getenv("RTLAMR_" + strings.ToUpper(f));
+		if 0<len(value) {
+			flag.Set(f, value)
```

Comment:
We need to handle errors produced here when values stored in environment variables aren't valid.